ci-operator/step-registry/stackrox/orion/stackrox-orion-commands.sh

```
@@ -0,0 +1,428 @@
+
+python --version
+pushd /tmp
+python -m virtualenv ./venv_qe
+source ./venv_qe/bin/activate
+
+if [[ $TAG == "latest" ]]; then
+    LATEST_TAG=$(git ls-remote --tags https://github.com/cloud-bulldozer/orion.git | awk -F'refs/tags/' '{print $2}' | grep -v '\^{}' | sort -V | tail -n1)
+else
+    LATEST_TAG=$TAG
+fi
+git clone -q --branch $LATEST_TAG $ORION_REPO --depth 1
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)